tsconfig.json

```diff
@@ -1,12 +1,16 @@
 {
+  "include": ["source/**/*.ts"],
+  "exclude": ["source/index.d.ts"],
```

Comment:
Do we need this `include` section at all?

Reply:
Without `include` sections code look like this `declare module "source/server"` instead of `declare module "server` and empty declarations will be generated from tests directory `declare module "tests/client" { }`

Reply:
OK, right. Then `"baseUrl": "source"` or `"rootDir": "source"` should be enough.